tmt/export/nitrate.py

```diff
@@ -2,6 +2,7 @@
 import os
 import re
 import types
+import urllib.parse
```

Comment:
can be just `from urllib.parse import urlparse`?

Reply:
It can, but I'd first agree on some policy. I for one am not a big friend of `import foo from bar` as I think it may easily pollute the namespace with dozens of names that have no place on the module level, but it does make sense for some common, well-known and well-understood names - `Enum`, `Guest`, `format`, `Result` and so on. If we could set up a linter check that prevents us from importing everything everywhere, I'd be much happier.

Reply:
Sure, I have no horse in the usage of `from` in external imports.     
Still, it could be `import urllib.parse.urlparse` though.

Reply:
Hmm, isn't `urlparse` a callable, not a module?

```
$ python
Python 3.12.6 (main, Sep  9 2024, 00:00:00) [GCC 13.3.1 20240522 (Red Hat 13.3.1-1)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import urllib.parse.urlparse
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'urllib.parse.urlparse'; 'urllib.parse' is not a package
>>> 
```

Reply:
ha, indeed. ok so I do have a horse :D